### DISCUSSION TITLE
Cluster without LB - endpoint IP

### DISCUSSION THREAD

**Discussion by a5r0n** [syself/cluster-api-provider-hetzner#274] [Q&A] [answered]
I'm just trying the new feature, 
And I got a validation error about endpoint IP.
The IP should be fetched from the API, no?

`ValidationError(HetznerCluster.spec.controlPlaneEndpoint): missing required field "port" in io.x-k8s.cluster.infrastructure.v1beta1.HetznerCluster.spec.controlPlaneEndpoint`

**Comment by batistein** [comment]
The error says that you need to specify in HetznerCluster.spec.controlPlaneEndpoint.port I guess right now this is missing? This should be 6443

**Comment by batistein** [comment, marked as answer]
And nothing is fetched - you need to specify a domain and configure the appropriate A records for your load balancer or control planes.

**Comment by janiskemper** [comment]
Sorry for not documenting this properly btw. It was on our list and we forgot it!
